docker/openproblems-python37-scgen/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM condaforge/mambaforge
+
+ENV DEBIAN_FRONTEND=noninteractive
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
+
+RUN \
+    apt-get update && \
+    apt-get install -qy sudo git unzip&& \
+    useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
+    chmod g+w /etc/passwd && \
+    echo "${NB_USER}    ALL=(ALL)    NOPASSWD:    ALL" >> /etc/sudoers && \
+    # Prevent apt-get cache from being persisted to this layer.
+    find /var/lib/apt/lists/ ! -type d -exec rm '{}' \;
```

Comment:
I believe this is the line that should be added to other dockerfiles to make more lightweight, no?

Reply:
I just went with this: ` && rm -rf /var/lib/apt/lists/*` now as it's recommended in the docker best practices